Make tags look better / spelling fix (#162)
PrismLauncher/bot: tags/java.md

---
embed:
  title: Java Instructions 
  color: orange
---

> For security reasons, Prism Launcher does not bundle in Java with itself. The instructions to setup and install Java can be found [here](https://prismlauncher.org/wiki/getting-started/installing-java/).

Reviewer: Security reasons isn't quite right.
```suggestion
Currently, Prism Launcher does not bundle Java with itself. The instructions to setup and install Java can be found [here](https://prismlauncher.org/wiki/getting-started/installing-java/).
```

> For security reasons, Prism Launcher does not bundle in Java with itself. The instructions to setup and install Java can be found [here](https://prismlauncher.org/wiki/getting-started/installing-java/).

Reviewer 2: I think this isn't because of "security reasons" but more because it isn't implemented yet?

> ***DO NOT USE ORACLE!!!***

Reviewer: why?

Reviewer: yeah, why the oracle part. I mean, I prefer FOSS JREs myself, but I don't think there is something inherently wrong with Oracle JDK specifically?

Author: I was going to type out a long story about my ever loving hatred for Oracle but then I realized now is not the time or place. Removing that part.

> ***DO NOT USE ORACLE!!!***

Reviewer 2: yes the webpage also states this, but is it really worth having this here?
instead it might be better to just directly poste the whole faq post from the webpage, but that's too much...

(non issue here) also the webpage post isn't the best for quickly figuring out where to get java...

Author: I can rework the webpage when I get home from school. I removed the Oracle bit since I've come to the conclusion that it ultimately doesn't matter.
